_includes/navigation.html

```diff
@@ -17,5 +17,8 @@
     <li>
       <a class="page-link" href="{{ '/volunteer' | prepend: site.baseurl }}">Volunteer</a>
     </li>
+    <li>
+      <a class="page-link" href="https://link.justgiving.com/v1/charity/donate/charityId/3118662?tipScheme=TipJar2.1&isRecurring=true&amount=20.00&reference=givingcheckout_tj21">Donate</a>
```

Comment:
Suggest pulling the url out to a configuration value, not because it'll change but so that we have a single constant reference rather than lots of literals. This will also make it easier to copy across into our sub-sites if we want.

There might also be an argument for having a redirect from, say, `/donate`, though that might be beyond what we need for now.

Reply:
I can actually see examples where it might change, e.g. if we decide to tweak the default values. Which would support the idea of having it in a variable.

/donate also sounds like a good idea, but like you say a separate task

Reply:
Done in 3c8899a :)